### PR TITLE
Allow to use i18n extensions for portal templates

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Allow to use i18ned templates extenions
# Impacts

Captive portal templates
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Allow to create different templates based on the browser locale

### CODE REVIEW

File: lib/pf/Portal/Profile.pm [hunk 1]
```diff
@@ -596,6 +597,28 @@ sub getRootModuleId {
...
+
+=cut
+
+sub getAupTemplate {
+    my ($self, $base_template) = @_;
+    my $locale = setlocale(POSIX::LC_MESSAGES);
+    my @parts = split(/\./, $base_template);
+    my $suffix = pop(@parts);
+    my $prefix = join('.', @parts);
+    if(defined($locale) && $locale =~ /([a-zA-Z]+)[_]{0,1}/) {
```
Comment (cgx) [inline lib/pf/Portal/Profile.pm]: Keep it simple: 

``` perl
if(defined($locale) && $locale =~ /([a-zA-Z]+)_?/) {
```

**Comment by cgx** [comment]
While you are at it, why not making all TT templates i18nable?

**Comment by julsemaan** [comment]
@cgx 

Done.

**Comment by fdurand** [comment]
Looks good to me.

**Pull request merged**